Cubical/Homotopy/HSpace.agda

```diff
@@ -27,6 +34,89 @@ record AssocHSpace {ℓ : Level} {A : Pointed ℓ} (e : HSpace A) : Type ℓ whe
                (μ-assoc (pt A) y z)
                refl
 
+record LeftInvHSpace {ℓ : Level} {A : Pointed ℓ} (e : HSpace A) : Type ℓ where
+  constructor LeftInvHSp
+  field
```

Comment:
Just a minor suggestion: You can ```open HSpace e``` here if you like and then use ```μ x``` directly below.
Also, constructors are almost never used since we encourage copattern definitions.